ShooterSubsystem.java

```diff
@@ -0,0 +1,91 @@
+package frc.robot.subsystems.shooter;

```

Comment:
Delete this file, you already have the correct one in the correct folder with newer code